src/languageProvider/locales/ko-KR.json

```diff
@@ -156,7 +155,6 @@
   "str.cancel": "취소",
   "str.changePassphrase": "암호 변경",
   "str.close": "닫기",
-  "str.confirm": "승인",
```

Comment:
this one is still used, i think we need to add it.

Reply:
the string was wrong before. it used to be `str.confirm: "Ok"`
I changed it to Confirm and so we need to re-translate.